.goreleaser.yaml

```diff
@@ -0,0 +1,59 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - main: ./cmd/stream-cli/
+    binary: stream-cli
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+
+checksum:
+  name_template: 'checksums.txt'
+
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
```

Comment:
might be nice to keep it

Reply:
actually, this is not being used. we still use the npm package to generate the changelog

Reply:
removing it from the config